src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -54,6 +54,20 @@
   <!-- Need to reference the Sonar Roslyn analyzers: we need to reference one of the assmblies,
        and embed all of them in the VSIX. -->
   <Import Project="..\EmbeddedSonarAnalyzer.props" />
+  <ItemGroup Label="Embedded SonarLint analyzers">
```

Comment:
This section was cut-and-paste without changes from the EmbeddedSonarAnalyzer.props file.